README.md

```diff
@@ -19,6 +19,22 @@ The Charts in the master branch of this repository match the latest packaged Cha
 
 The purpose of this repository is to provide a place for maintaining and contributing official Charts, with CI processes in place for managing the releasing of Charts into the Chart Repository.
 
+The Charts in this repository are organized into two folders:
+* stable
+* incubator
+
+Stable Charts:
+
+1. Provide a method for data persistence
```

Comment:
Should we add `(if applicable)`, since there are a few that might have no use for persistence at all (like memcached)